.github/actions/ktlint/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM openjdk:11.0.1-jdk-slim-stretch
+
+RUN apt-get update && apt-get -y install curl
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Delete the apt-get lists after installing something](https://app.codacy.com/manual/davidmerrick/AdventOfCode2019/pullRequest?prid=5020686)